src/EasyAbp.Abp.Trees.EntityFrameworkCore/Microsoft/Extensions/DependencyInjection/AbpTreesServiceCollectionExtensions.cs

```diff
@@ -17,7 +14,7 @@ public static class AbpTreesServiceCollectionExtensions
             //TODO: Custom option action
             //optionsBuilder?.Invoke(options);
 
-            new EfCoreTreeRepositoryRegistrar(options).AddRepositories();
```

Comment:
This is not typo,ref from [abp](https://github.com/abpframework/abp/blob/42f37c5ff01ad853a5425d15539d4222cd0dab69/framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/EntityFrameworkCore/DependencyInjection/EfCoreRepositoryRegistrar.cs)
I will keep `EfCoreTreeRepositoryRegister`